careers/jobs/senior-fullstack-node-engineer.md

```diff
@@ -0,0 +1,65 @@
+# Senior Fullstack Node Engineer
+
+## About DVELP
+
+DVELP is a remote-first London based software development consultancy.
+
+We work with leading brands like M&S, British Airways, Google, and Twilio helping them to build secure, robust platforms for highly sensitive data.
+
+Our experience ranges from designing and delivering end-to-end digital banking solutions, through to bespoke internal communication tools for the hospitality sector.
+
+We have a strong engineering background, which remains at the centre of everything we do.
+
+We are currently hiring for a Senior Full-stack Node Developer to join our team, initially to work with one of our large established London based clients. We are looking for like minded engineers, who are a master of their craft and with experience in leading a team.
```

Comment:
`We are currently hiring for a Senior Full-stack Node Developer to join our team` -> `We are currently hiring for a Senior Full-stack Node Developer to join our team`.

Reply:
There wasn't a change here - did you mean to change `hiring` for `looking` (as that's what I've done anyway)?